src/components/graph/graph.helper.js

```diff
@@ -102,7 +102,7 @@ function _initializeLinks(graphLinks, config) {
  * and highlighted values.
  * @memberof Graph/helper
  */
-function _initializeNodes(graphNodes) {
+export function _initializeNodes(graphNodes) {
```

Comment:
Please remove the `_` from the function name here. It's used to denote a non-exported function. Also please add the `initializeNodes` function to the `export` block at the end of the final instead of having the `export` inlined here (just for consistency)